packages/dm-core/src/components/EntityView.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+
+import styled from 'styled-components'
+import { CircularProgress } from '@equinor/eds-core-react'
+import { ErrorBoundary } from '../utils/ErrorBoundary'
+import { useRecipe } from '../hooks'
+import { IUIPlugin } from '../types'
+
+const Wrapper = styled.div`
+  align-self: start;
+  justify-content: space-evenly;
+  width: 100%;
+`
+
+export const EntityView = (props: IUIPlugin): JSX.Element => {
+  const { idReference, type, onSubmit, onOpen } = props
+  const { recipe, isLoading, error, getUiPlugin } = useRecipe(type)
+
+  if (isLoading)
+    return (
+      <div style={{ alignSelf: 'center', padding: '50px' }}>
+        <CircularProgress color="primary" />
+      </div>
+    )
```

Comment:
Could use the Loading.tsx component instead

Reply:
fixed